CMakeLists.txt

```diff
@@ -71,5 +67,5 @@ if (SUPPORT_MID_TIMIDITY)
 endif()
 
 target_include_directories(SDL2_mixer PUBLIC include)
-target_link_libraries(SDL2_mixer PRIVATE SDL2)
+target_link_libraries(SDL2_mixer PRIVATE ${SDL2_LIBRARIES})
```

Comment:
append ` ${SDL2_LIBRARY}` here for better compatibility.

Reply:
And also please revert the `#include` path changes and squash the commits for easier review.